Components/Rating/Rating.css

```diff
@@ -1,8 +1,4 @@
-/*
-=====
-DEPENDENCES
-=====
-*/
+
 
 .screen-reader{
     width: var(--screenReaderWidth, 1px) !important;
```

Comment:
if possible, change  the specificity of the css property instead of using !important.